packages/app-root/server/server.js

```diff
@@ -0,0 +1,53 @@
+if (process.env.NEWRELIC_LICENSE_KEY) {
```

Comment:
The express server runs on `yarn start` too, but we need to test that on Kubernetes. 

Your comment reminds me that the live content pages app isn't logging to New Relic either.

Reply:
Right! I think that's what I'm getting at. The content pages app will probably never use a custom express server in deployment again, so I New Relic should not be setup for app-root either. Thoughts?

Reply:
How do we debug slow pages and Postgres queries without New Relic?

Reply:
I mean, content pages should be using New Relic in production too. It should be imported when the Next app loads and starts, at the top of `next.config.js`.

Reply:
Does the debugging via New Relic happen via logging from a production deploy, a local build, or both?

Reply:
🤔 app-content does have New Relic imported into `next.config.js` but app-content-pages does not.

Reply:
Any environment that has `NEWRELIC_LICENSE_KEY` set, according to the code. That’s production and staging deploys.

Reply:
Looks like they only have examples for the pages router. 
https://github.com/newrelic-experimental/newrelic-nextjs-integration

Reply:
In addition to lack of New Relic + App Router documentation, I strongly suggest that New Relic isn't included in app-root (for now). We don't really know what deployment of this app will look like yet, and I'd prefer to setup logging tools that are used in staging + production envs when we get to that step. I've added it as a to-do in the [project board](https://github.com/orgs/zooniverse/projects/43).

Otherwise everything here is looking good!